packages/text-area/src/vaadin-text-area-mixin.js

```diff
@@ -178,6 +190,21 @@ export const TextAreaMixin = (superClass) =>
       inputField.scrollTop = scrollTop;
     }
 
+    /** @private */
+    __updateMinHeight(minRows) {
+      if (!this._inputField || !this.inputElement) {
+        return;
+      }
+
+      // For minimum height, just set the number of rows on the native textarea,
+      // which causes the input container to grow as well.
+      // Do not override this on custom slotted textarea as number of rows may
+      // have been configured there.
+      if (this.inputElement === this.__textAreaController.defaultNode) {
```

Comment:
We have provided workarounds to use a custom slotted textarea to allow customizing the min. number of rows, and I have at least used this once when working on Control Center. I figure we should not break that and only propagate to `rows` when not using a custom textarea element.

Alternatively, we could change the implementation to calculate a `min-height` for the textarea based on line height and padding. We'll likely need something similar for implementing `max-rows` anyway.

Reply:
An alternative could also be to not have a default value for the `min-rows` property and not set `rows` as long as there is no explicit value. Then the default behavior would be the same as before.

Reply:
I'm ok with not updating slotted `<textarea>` but it probably could be documented in the property JSDoc.

Reply:
Added a note.